justfile

```
@@ -4,12 +4,15 @@ build:
   echo "Building for local arch"
 
   export VERSION="${RELEASE_TAG_VERSION:-dev}"
+  export DATETIME=$(date -u +"%Y-%m-%d %H:%M:%S")
 
   go build \
     -ldflags="-s -w" \
     -ldflags="-X 'github.com/open-sauced/pizza-cli/pkg/utils.Version=${VERSION}'" \
     -ldflags="-X 'github.com/open-sauced/pizza-cli/pkg/utils.Sha=$(git rev-parse HEAD)'" \
-    -o build/pizza
+    -ldflags="-X 'github.com/open-sauced/pizza-cli/pkg/utils.Sha=$(git rev-parse HEAD)'" \
+    -ldflags="-X 'github.com/open-sauced/pizza-cli/pkg/utils.Datetime=${DATETIME}'" \
+    -o build/pizza-${GOOS}-${GOARCH}
```

The `-${GOOS}-${GOARCH}` wasn't there previously. I added it, but not sure why it wasn't present fpr this particular instance. Is this because it's only if someone builds locally?

---

That seems to be the case as running `just install` after running `just build` works again after removing the `-${GOOS}-${GOARCH}` I added.

---

Looks like `-o build/pizza-${GOOS}-${GOARCH}` made it back in for the local short circuit `just build`

```suggestion
    -o build/pizza
```

---

How did that sneak back in? 🫠